.github/workflows/test.yml

```diff
@@ -74,6 +74,6 @@ jobs:
       - name: Install
         if: steps.yarn-cache.outputs.cache-hit != 'true'
         run: CYPRESS_INSTALL_BINARY=0 yarn install
-      - run: yarn build-all
+      - run: CI=false yarn build
```

Comment:
This is a bit unfortunate, but this prevents the build from failing if there are warnings, and there are a lot of warnings from our dependencies. I imagine these warnings will get fixed over time as libs update for Webpack v5.